scripts/llnl_scripts/llnl_lc_build_tools.py

```diff
@@ -460,50 +471,59 @@ def full_build_and_test_of_tpls(builds_dir, job_name, timestamp):
     uberenv_create_mirror(prefix,mirror_dir)
     # write info about this build
     write_build_info(pjoin(prefix,"info.json"), job_name)
+
+    repo_dir = get_repo_dir()
+    # Clean previously generated host-configs into TPL install directory
```

Comment:
I will after I am done finishing the workflow.

Reply:
Because this is shared (at least going to) with the spack repo, I needed it to be in a shared/easy location to access.  The base of the source repository seemed like a good location for it.  The generated host-configs get generated and used at the base of source then copied into the shared LLNL TPL directory by build_tpl.py.  I felt it was a good idea to leave them at the base of the repo so its easy to copy them into the host-configs directory.